### ISSUE TITLE
General : Class name not perfect for all js file

### ISSUE THREAD

**Issue by ashokdudhat** [wpfoodmanager/wp-food-manager#398]
Please check the boilerplate and follow according to class name.

<img width="1321" alt="Screenshot 2023-03-26 at 18 29 17" src="https://user-images.githubusercontent.com/15089059/227789963-eb4055c3-75e7-4297-844e-88bde1224ed4.png">

### JS Class name
 
We are using **Singleton pattern** practice to create Javascript file.

Always use prefix **WPFM** to every js class name with following camelcase style.

For Core Plugin :  General Rule :    `WPFM_ClassPurpose` (CamelCaseBasedOnFileName)
For Add on :  General Rule :    `WPFM_Addon_Name_ClassPurpose` (CamelCaseBasedOnFileName)

**For Example**

File name : https://github.com/wpfoodmanager/wp-food-manager/assets/js/food-dashboard.js

`WPFM_FoodDashboard`

File Name : https://github.com/wpfoodmanager/wp-food-manager/assets/js/food-submission.js

`WPFM_FoodSubmission`

**This is wrong :**  WPFM_Foodsubmission. (IT IS NOT CamelCase)

If you are creating **js class file for add on of WP Food Manager** then please use add on name after the prefix. For example Online Order Frontend js class file.

`WPFM_Online_Order_Frontend`

### In short for us one formula : 

**Every class name must start with prefix  "WPFM_" . Doesn't it is php class file or js class file (singleton pattern).**

**Comment by vry-kvl** [comment]
@ashokdudhat sir, Now for all js class files name starts with the prefix "WPFM_" and with singleton pattern and camelcase like :

- WPFMAdminsettings -> WPFM_AdminSettings
- WPFMAdmin -> WPFM_Admin

@krinaydhanani - Please review it.

**Comment by krinaydhanani** [comment]
This issue was fixed. I am going to close this issue.
